src/popup/locales/en.json

```diff
@@ -39,6 +39,19 @@
     "required": {
       "msg": "Please fill in all fields."
     },
+    "payload-info": {
+      "title": "Payload message",
+      "msg": "A payload is an optional public message composed by the sender and sent to the recipient along with a blockchain transaction.",
+      "desc": "Payload can be utilized as additional transaction identifier, destination tag (required by some centralized exchanges when depositing funds), or just to provide any extra information in free-form text."
+    },
+    "payload-form": {
+      "title": "Add payload (optional)",
+      "desc": "Send a public message with your transaction for approx. 0.00000002 AE per character.",
+      "label": "Payload message",
+      "error-message": "Your message exceeds the limit of 280 characters",
+      "done-btn": "Done",
+      "cancel-btn": "Cancel"
+    },
```

Comment:
I know that our translation files are not consistent in terms of the naming (kebab-case vs camelCase), but at some point we will need to fix this.In my opinion the `camelCase` is the goal for us. So I suggest to use camelCase for new code added to the translation files.